lib/src/Pos/Web/Api.hs

```diff
@@ -65,28 +65,27 @@ type HealthCheckApi =
 baseNodeApi :: Proxy BaseNodeApi
 baseNodeApi = Proxy
 
--- | GodTossing specific API.
-type GodTossingApi =
+-- | SSC specific API.
+type SscApi =
     "toggle"
         :> Capture "enable" Bool
         :> Post '[JSON] ()
     -- :<|>
     -- "has_secret" :> Get '[JSON] Bool :<|>
     -- "secret" :> Get '[JSON] SharedSeed :<|>
-    -- "stage" :> Get '[JSON] GodTossingStage
+    -- "stage" :> Get '[JSON] SscStage
 
 -- | Helper Proxy.
-godTossingApi :: Proxy GodTossingApi
-godTossingApi = Proxy
+sscApi :: Proxy SscApi
+sscApi = Proxy
 
--- | Servant API which provides access to full node internals with
--- GodTossing SSC.
-type GtNodeApi =
+-- | Servant API which provides access to full node internals with SSC.
+type SscNodeApi =
     BaseNodeApi
     :<|>
     "god_tossing"
```

Comment:
Also there is no need to prefix `ssc` api with `god_tossing`.
Don't worry about breaking things, this server is just a swalka which is not used for anything significant.